benchmark/browser/README.md

```diff
@@ -14,42 +14,42 @@ You should use these numbers exclusively for comparing performance between diffe
 
 ```
 noop (baseline):
-mean: 4.79ms, median: 4.75ms
+mean: 4.96ms, median: 4.56ms
 -------------
 React primitives:
-mean: 44.62ms, median: 44.57ms
+mean: 39.20ms, median: 39.14ms
 -------------
 React components:
-mean: 55.43ms, median: 54.68ms
+mean: 48.49ms, median: 47.61ms
 -------------
 Styled Material-UI:
-mean: 80.52ms, median: 80.97ms
+mean: 72.32ms, median: 70.97ms
```

Comment:
I start to wonder if we shouldn't display the duration relative to a baseline in %. I mean, as a way to make independent from the CPU power available on the machine that runs them. It's hard to tell in which direction we are moving.

Reply:
I had the same issue when running the benchmarks these days... I was trying to see relative differences between the different examples, but even that doesn't help as there are outlines in a few of the examples even on the same run... Because of the second issue, I don't think that even if we display the duration relative to the baseline it would help much... My next step would be to see if the `React.Profiler` will give more stable numbers..